Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "html-build"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tokio = { version = "1", features = ["full"] }
+html5ever = "*"
+markup5ever_rcdom = "*"
```

Comment:
We should probably pin these to their current versions? We can hopefully rely on dependabot to update them over time.